### PR TITLE
[FrameworkBundle] Use default_locale as default value for translator.fallbacks

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | yes 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

Simplify translator's config:

Before:

```yaml
    default_locale: fr
    translator:
        default_path: '%kernel.project_dir%/translations'
        fallbacks:
            - fr
```

After:

```yaml
    default_locale: fr
    translator:
        default_path: '%kernel.project_dir%/translations'
```

### CODE REVIEW

**Review by azjezz — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Review by ogizanagi — APPROVED** [review]

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -763,9 +763,10 @@ private function addTranslatorSection(ArrayNodeDefinition $rootNode)
                     ->fixXmlConfig('path')
                     ->children()
                         ->arrayNode('fallbacks')
+                            ->info('Defaults to the value of "default_locale".')
                             ->beforeNormalization()->ifString()->then(function ($v) { return [$v]; })->end()
                             ->prototype('scalar')->end()
-                            ->defaultValue(['en'])
+                            ->defaultValue([])
```
Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php]: Isn't this a BC break?

Comment (nicolas-grekas) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php]: Not if default_local was set to the same value, right? Should always be the case?

**Review by nicolas-grekas — APPROVED** [review]

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php (hunk 1 above)
Comment (xabbuh) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php]: I don't think so. For a German site I would set `default_locale` to `de` but keep the fallback set to `en` if my team writes English messages and translations are provided later by a separate team.

Comment (dunglas) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php]: It's an edge case that can be fixed very easily, and deserves to be explicit IMHO. The current behavior create a WTF situation where you change the default_locale, but it's not taken into account by Symfony itself without having to change another setting.

**Comment by fabpot** [comment]
Thank you @dunglas.

**Pull request merged**